configure.ac

```diff
@@ -5967,6 +5976,7 @@ echo "   * DH:                         $ENABLED_DH"
 echo "   * DH Default Parameters:      $ENABLED_DHDEFAULTPARAMS"
 echo "   * ECC:                        $ENABLED_ECC"
 echo "   * ECC Custom Curves           $ENABLED_ECCCUSTCURVES"
+echo "   * ECC Weak Curves             $ENABLED_ECCWEAKCURVES"
```

Comment:
I can't see this being set.
Is the eccminsz meant to set this?

Reply:
Fixed. Was from older cold that was later changed. Good catch